securedrop/tests/test_journalist.py

```diff
@@ -559,6 +556,41 @@ def test_admin_add_user_too_short_username(self):
                                           is_admin=False))
         self.assertIn('Invalid username', resp.data)
 
+    def test_admin_sets_user_to_admin(self):
+        self._login_admin()
+        new_user = 'admin-set-user-to-admin-test'
+        resp = self.client.post(url_for('admin_add_user'),
+                                data=dict(username=new_user,
+                                          password=VALID_PASSWORD,
+                                          is_admin=False))
+        assert resp.status_code in (200, 302)
+        journo = Journalist.query.filter(Journalist.username == new_user).one()
```

Comment:
assert journo.is_admin is False

Would be nice to have since it shows we go from one state to another.